File: python/border.py
import requests
import json
# Generate a border with a desired color for a given image.
# https://pixlab.io/#/cmd?id=border for more info.
req = requests.get('https://api.pixlab.io/border',params={
  'img':'https://www.allaboutbirds.org/guide/PHOTO/LARGE/blue_jay_8.jpg',
  'width':20,
  'color':'yellow',
  'key':'My_PixLab_Key'
 })
reply = req.json()
if reply['status'] != 200:
	print (reply['error'])
else:
    print ("Link to the pic: "+ reply['link'])

File: java/Border.java
import java.io.IOException;

import org.json.JSONException;
import org.json.JSONObject;

import okhttp3.HttpUrl;
import okhttp3.OkHttpClient;
import okhttp3.Request;
import okhttp3.Response;

public class Border {
	// Generate a border with a desired color for a given image.
	// https://pixlab.io/#/cmd?id=border for more info.
	
	// Target image
	private static String img = "https://www.allaboutbirds.org/guide/PHOTO/LARGE/blue_jay_8.jpg";
	// Your PixLab key
	private static String key = "Pix_Key";
	
    static OkHttpClient client = new OkHttpClient();

	public static void main(String[] args) throws IOException, JSONException {
		
		HttpUrl httpUrl = new HttpUrl.Builder()
                .scheme("https")
                .host("api.pixlab.io")
                .addPathSegment("blur")
                .addQueryParameter("img", img)
                .addQueryParameter("key", key)
                .addQueryParameter("width", "20")
                .addQueryParameter("color", "yellow")
                .build();
		
		Request requesthttp = new Request.Builder()
                .addHeader("accept", "application/json")
                .url(httpUrl)
                .build();

        Response response = client.newCall(requesthttp).execute();

		JSONObject jResponse = new JSONObject(response.body().string());
		if (jResponse.getInt("status") != 200) { 
			System.out.println("Error :: " + jResponse.getString("error"));
			System.exit(1);
		}else {
			System.out.println("Link to the picture: "+ jResponse.getString("link"));
		}
	}

}
